Task1/Task4/BitmapAdapter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AForge.Imaging.Filters;
+using System.Drawing;
+
+namespace Task4
+{
+    public class BitmapAdapter : INegative
+    {
+        private Bitmap image;
+
+        public BitmapAdapter(Bitmap image)
+        {
+            this.Image = image;
```

Comment:
Anya, install Resharper on your station. If you have it already installed, pay attention to its tips.